### PR TITLE
Refactor measure definitions for readability

### PR DESCRIPTION
**Pull request by richiecroker**
Refactor measure definitions to include MDR name from pseudo-BNF as commentary. This will help when reviewing old measures.

### CODE REVIEW

File: openprescribing/frontend/management/commands/measure_definitions/ace.json [hunk 1]
```diff
@@ -29,13 +29,145 @@
...
+    "'0205051W0AAACAC',  --Imidapril HCl_Tab 20mg \n",
+    "'0205051W0BBAAAA',  --Tanatril_Tab 10mg \n",
+    "'0205051W0BBABAB',  --Tanatril_Tab 5mg \n",
+    "'0205051W0BBACAC',  --Tanatril_Tab 20mg \n",
+    "'0205051Y0AAAAAA',  --Perindopril Arginine_Tab 2.5mg \n",
+    "'0205051Y0AAABAB',  --Perindopril Arginine_Tab 5mg \n",
+    "'0205051Y0AAACAC',  --Perindopril Arginine_Tab 10mg \n",
+    "'0205051Y0BBAAAA',  --Coversyl Arginine_Tab 2.5mg \n",
+    "'0205051Y0BBABAB',  --Coversyl Arginine_Tab 5mg \n",
+    "'0205051Y0BBACAC',  --Coversyl Arginine_Tab 10mg"
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/measure_definitions/ace.json]: Extra comma, missing closing paren

File: openprescribing/frontend/management/commands/measure_definitions/ace.json [hunk 2]
```diff
@@ -29,13 +29,145 @@
...
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/measure_definitions/ace.json]: Ditto

File: openprescribing/frontend/management/commands/measure_definitions/ace.json [hunk 3]
```diff
@@ -29,13 +29,145 @@
...
+    "'0205051W0AAACAC',  --Imidapril HCl_Tab 20mg \n",
+    "'0205051W0BBAAAA',  --Tanatril_Tab 10mg \n",
+    "'0205051W0BBABAB',  --Tanatril_Tab 5mg \n",
+    "'0205051W0BBACAC',  --Tanatril_Tab 20mg \n",
+    "'0205051Y0AAAAAA',  --Perindopril Arginine_Tab 2.5mg \n",
+    "'0205051Y0AAABAB',  --Perindopril Arginine_Tab 5mg \n",
+    "'0205051Y0AAACAC',  --Perindopril Arginine_Tab 10mg \n",
+    "'0205051Y0BBAAAA',  --Coversyl Arginine_Tab 2.5mg \n",
+    "'0205051Y0BBABAB',  --Coversyl Arginine_Tab 5mg \n",
+    "'0205051Y0BBACAC'  --Coversyl Arginine_Tab 10mg)"
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/measure_definitions/ace.json]: This closing paren is commented out. It needs to be on its own line

Comment (richiecroker) [reply openprescribing/frontend/management/commands/measure_definitions/ace.json]: fixed

File: openprescribing/frontend/management/commands/measure_definitions/statinintensity.json [hunk 4]
```diff
@@ -29,20 +29,39 @@
   ],
   "numerator_from": "{hscic}.normalised_prescribing_standard ",
   "numerator_where": [
-    "(bnf_code like '0212000AA%' or bnf_code like '0212000AC%' or bnf_code like '0212000B0%' ",
-    "or bnf_code like '0212000C0%' or bnf_code like '0212000M0%' or bnf_code like '0212000X0%' ",
-    "or bnf_code like '0212000Y0%') AND ",
-    "(bnf_code not like '0212000Y0%AH' and bnf_code not like '0212000B0%AB' and bnf_code not like '0212000B0%AN' ",
-    "and bnf_code not like '0212000B0%AC' and bnf_code not like '0212000B0%AP' and bnf_code not like '0212000B0%AD' ",
-    "and bnf_code not like '0212000AA%AA' and bnf_code not like '0212000AA%AB' and bnf_code not like '0212000AA%AC')"
+  "bnf_code LIKE '0212000AA%' OR --Rosuvastatin Calcium \n",
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/measure_definitions/statinintensity.json]: Missing `(`

File: openprescribing/frontend/management/commands/measure_definitions/glaucoma.json [hunk 5]
```diff
@@ -27,15 +27,24 @@
...
   "denominator_from": "{hscic}.normalised_prescribing_standard ",
   "denominator_where": [
-       "(bnf_code LIKE '1106000%') AND ",
-    "(bnf_code not like '1106000B0%' AND bnf_code not like '1106000C0%')"
+    "(",
+    "bnf_code LIKE '1106000%') --Treatment of Glaucoma (branded and generic) \n",
+    "AND ",
+    "(",
+    "bnf_code NOT LIKE '1106000B0B%' AND --Acetazolamide (branded and generic) \n",
+    "bnf_code NOT LIKE '1106000C0B%' --Acetazolamide (Parent) (branded and generic)",
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/measure_definitions/glaucoma.json]: Now you're missing a newline here

**Comment by sebbacon** [comment]
For the record, the code @richiecroker used to compare new measures (in staging bigquery) with old measures (in production query) is [here](https://nbviewer.jupyter.org/github/ebmdatalab/jupyter-notebooks/blob/master/SQL_check_new_measures/Check%20new%20measure%20SQL.ipynb).

**Pull request merged**